### ISSUE TITLE
Knowledge transmutation tablet remove bug

### ISSUE THREAD

**Issue by nitsuga5124** [sinkillerj/ProjectE#675]
http://imgur.com/C9l1uib,fkmiuno,Av5RzFR#0

1: i have 3 items in the table
2: i add emc to something
3: i lose all the knowledge

fixit

i have ProjectE 1.4.5

**Comment by MaPePeR** [comment]
Don't open your transmutation table before the emc mapping is finished. duplicate of #657
